components/floatingactionbutton/overview.md

```diff
@@ -0,0 +1,77 @@
+````
+
+## Position and Alignment
+
+You can control how the Floating Action Button is positioned relative to its associated container. Read more about the [Blazor Floating Action Button positioning...](slug://fab-positions)
+
+## Events
+
+The Blazor Floating Action Button fires events that you can handle and respond to user actions. [Read more about the Blazor Floating Action Button events...](slug://fab-events).
+
+## Floating Action Button Parameters
+
+The Blazor Floating Action Button provides various parameters that allow you to configure the component. Also check the [Floating Action Button public API](/blazor-ui/api/Telerik.Blazor.Components.TelerikFloatingActionButton).
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Parameter | Type and Default Value | Description |
+|----------|----------|----------|
+| `Enabled` | `bool` <br/> (`true`) | Whether the Floating Action Button is enabled. |
+| `Id` | `string` | The `id` attribute of the Floating Action Button. |
+| `Icon` | `object` | The icon rendered in the Floating Action Button. Can be set to a predefined Telerik icon or a custom one. | 
+| `Title` | `string` | The `title` attribute of the Floating Action Button. |
+| `PositionMode` | `FloatingActionButtonPositionMode ` <br /> (`Fixed`) | The position of the Floating Action Button relative to the container. |
+| `HorizontalAlign` | `FloatingActionButtonHorizontalAlign` <br /> (`End`) | Determines if the left or the right side of the Floating Action Button will touch its parent container. [Read more about Floating Action Button positioning.](slug://fab-positions) |
+| `VerticalAlign` | `FloatingActionButtonVerticalAlign` <br /> (`Bottom`) | Determines if the Floating Action Button will touch the parent container with its top or bottom side. |
+| `HorizontalOffset` | `string` <br /> (`16px`) | The horizontal offset value added to the button position, creating a blank space between the button and the parent. |
+| `VerticalOffset` | `string` <br /> (`16px`) | The vertical offset value added to the button position, creating a blank space between the button and the parent. |
+
+### Styling and Appearance
+
+The following parameters enable you to customize the appearance of the Blazor Floating Action Button:
+
+| Parameter | Type | Description |
+| --- | --- | --- |
+| `Class` | `string` | Defines the `<div class="k-badge">` CSS class rendered on the main wrapping element of the Floating Action Button component. Use it for [styling customizations](slug://themes-override). |
+| `Rounded` | `Telerik.Blazor.ThemeConstants.Badge.Rounded.Full` | Defines how rounded the borders of the Floating Action Button are. |
+| `Size` | `Telerik.Blazor.ThemeConstants.Badge.Size.Medium` | Sets the size of the Floating Action Button. |
+| `ThemeColor` | `Telerik.Blazor.ThemeConstants.Badge.ThemeColor.Primary` | Adjusts the background color of the Floating Action Button. |
```

Comment:
Using these long predefined values can make the table look strange. Consider removing these here and link to the Appearance article. Or, use something like these:

```suggestion
| `Rounded` | `string` <br /> `"full"` | Defines how rounded the borders of the Floating Action Button are. Use predefined value constants from the static class `Telerik.Blazor.ThemeConstants.Badge.Rounded`. |
| `Size` | `string` <br /> (`"md"`) | Sets the size of the Floating Action Button. Set it to a predefined value constant from the static class `Telerik.Blazor.ThemeConstants.Badge.Size`. |
| `ThemeColor` | `string` <br /> (`"primary"`) | Adjusts the background color of the Floating Action Button. Use predefined values from the static class `Telerik.Blazor.ThemeConstants.Badge.ThemeColor`. |
```